packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
+import { SxStyleProp } from '@kodiak-ui/core'
+import { Box } from '../Box'
+
+export type AvatarProps = {
+  imageSrc?: string
+  text: string
+  sx?: SxStyleProp
+}
+
+export function Avatar({ imageSrc, text, sx = {} }: AvatarProps) {
+  return imageSrc ? (
+    <Image src={imageSrc} sx={{ borderRadius: 'full', ...sx }} alt={text} />
+  ) : (
+    <Box
+      sx={{
+        borderRadius: 'full',
+        display: 'inline-flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        ...sx,
+      }}
+    >
+      {text}
+    </Box>
```

Comment:
I'm not sure that it is a good idea to add a default avatar with any text allowed. That's going to be a recipe for broken avatars if a developer is not expecting that. I would rather the Avatar component only care about rendering an actual image. 

If we need to default to a circle with text for a shop that doesn't have an image, we should do that in the parent context:

```
{shop?.logo ? <Avatar src={shop?.logo} /> : <Svg>{// generated SVG icon}</Svg>}
```

This will require another `Svg` component in Kodiak UI but I think it will be a better API in the long run where the Avatar renders only images.

Reply:
yup i felt uneasy about the `text=` prop too, which i raised in https://skyverge.slack.com/archives/C0138FU1XEX/p1590680995135900